packages/libs/src/sdk/services/Solana/programs/ClaimableTokensClient/ClaimableTokensClient.ts

```diff
@@ -275,10 +275,15 @@ export class ClaimableTokensClient {
               })
             }
           }
-        } catch (e) {
-          if (!(e instanceof ClaimableTokensError)) {
+        } catch (nestedError) {
+          if (nestedError instanceof ClaimableTokensError) {
+            throw nestedError
+          } else {
             // If failed to provide user friendly error, surface original error
```

Comment:
Note: This comment seems misleading as we are eating the error here and just console.warn-ing it.

Reply:
It actually rethrows the og error! See the outer scope

Reply:
Oh oops! My bad, carry on 😅